components/feature/webcompat-reporter/src/main/assets/extensions/webcompat-reporter/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "WebCompat Reporter",
   "description": "Report site compatibility issues on webcompat.com",
   "author": "Thomas Wisniewski <twisniewski@mozilla.com>",
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Comment:
Yes, let's always change the version when code changes. We're using GV's `ensureBuiltIn` now to install built-in extensions. It checks for version changes.

Reply:
Good to know, thanks! Will make sure this keeps getting bumped in the future.